pkg/aws/ec2/ec2.go

```diff
@@ -203,6 +206,17 @@ func parseENI(iface *ec2_types.NetworkInterface, vpcs ipamTypes.VirtualNetworkMa
 		}
 	}
 
+	for _, prefix := range iface.Ipv4Prefixes {
+		ips, e := ipPkg.PrefixToIps(aws.ToString(prefix.Ipv4Prefix))
+		err = e
+		if err != nil {
+			err = fmt.Errorf("unable to parse CIDR %s", *prefix.Ipv4Prefix)
```

Comment:
nit: To avoid accidental nil pointer de-referencing here, I would use `aws.ToString(prefix.Ipv4Prefix)` in the loop body here

Reply:
Missed this, removed it from couple other places too.